### DISCUSSION TITLE
ToDo mode change side

### DISCUSSION THREAD

**Discussion by rokn** [rxhanson/Rectangle#1045] [Ideas]
It would be nice to have the ToDo mode(which is great) to be placed on the left side as well as on the right. That's because if I have my laptop on the right of the main display I'd like to not go through the ToDo window.

**Comment by rxhanson** [comment]
Thanks for the feedback. At the moment, this is only in Rectangle Pro. I was planning on pulling it into Rectangle, but it hasn't bubbled up to top priority.

**Comment by rokn** [comment]
Hey it seems this was added in the latest version, thanks!

**Reply by rxhanson** [reply]
👍 This was the work of @decodism

**Comment by rokn** [comment]
Your comment about Rectangle Pro made me look it up and that window throw feature is a killer, i'll definitely be buying it after the trial. This kind of innovative features are what i'm looking for in this kind of tools not just the same old things

**Reply by rxhanson** [reply]
Glad you like it! You might also like [Charmstone](https://charmstone.app).
